pkg/executor/revoke.go

```diff
@@ -426,7 +426,7 @@ func (e *RevokeExec) revokeColumnPriv(ctx context.Context, internalSession sessi
 
 	if checkTablePrivTbl {
 		sql := strings.Join([]string{"SELECT * FROM %n.%n WHERE User=%? AND Host=%? AND DB=%? AND Table_name=%? AND Column_priv LIKE '%%", priv.Priv.String(), "%%';"}, "")
-		exists, err := recordExists(internalSession, sql, mysql.SystemDB, mysql.ColumnPrivTable, user, host, e.Level.DBName, e.Level.TableName)
+		exists, err := recordExists(internalSession, sql, mysql.SystemDB, mysql.ColumnPrivTable, user, host, dbName, e.Level.TableName)
```

Comment:
I think this PR should only contain log changes. Restore the scene as much as possible is more helpful in locating problems